dashboard/app/controllers/sections_controller.rb

```diff
@@ -8,6 +8,26 @@ def new
     redirect_to '/home' unless params[:loginType] && params[:participantType]
   end
 
+  def edit
+    return head :forbidden unless current_user&.admin
+
+    redirect_to '/home' unless params[:loginType] && params[:participantType]
```

Comment:
I don't think this is needed for edit section. `loginType` and `participantType` should already be set for existing sections

Reply:
Agreed.  Getting rid of this will cause some issues for another file expecting the param (specifically, the `useEffect` in `CurriculumQuickAssign` I am going to make a ticket for this for us to address separately.

Reply:
Noting that this is now https://codedotorg.atlassian.net/browse/TEACH-447